src/components/Navbar/index.view.tsx

```diff
@@ -1,8 +1,27 @@
 import React from "react";
+import Button, { ButtonProps } from "components/Button/index.view";
 import "./Navbar.scss";
 
-const Navbar: React.FC = () => {
-  return <div className="Navbar">Hello from Navbar</div>;
+interface NavbarProps {
+  title: string;
+  sponsorButtonProp: ButtonProps;
+  hackerButtonProp: ButtonProps;
+}
+
+const Navbar: React.FC<NavbarProps> = ({
+  title,
+  sponsorButtonProp,
+  hackerButtonProp,
+}: NavbarProps) => {
+  return (
+    <div className="Navbar">
+      <div className="Navbar__title">{title}</div>
+      <div className="Navbar__buttonContainer">
+        <Button {...sponsorButtonProp} />
+        <Button {...hackerButtonProp} />
```

Comment:
```suggestion
         {buttonProps.map((entry) => (
          <Button key={entry.label} {...entry} />
        ))}
```
Again, just to illustrate how powerful having an array of objects is. We were able condense a ton of code and make it way easier to add in changes.